content/guides/output-management.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: Output Management
-sort: 4
```

Comment:
Sorry for that, I read the guide so I followed, may I just remove these lines and commit?